module/Olcs/src/Form/Model/Fieldset/CloseConversationActions.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Olcs\Form\Model\Fieldset;
+
+use Laminas\Form\Annotation as Form;
+
+/**
+ * @codeCoverageIgnore Auto-generated file with no methods
+ * @Form\Name("main")
+ */
+class CloseConversationActions
+{
+    /**
+     * @Form\Attributes({
+     *     "type":"submit",
+     *     "data-module": "govuk-button",
+     *     "class": "govuk-button govuk-button--warning",
+     *     "id": "close",
+     * })
+     * @Form\Options({
+     *     "label": "End and archive conversation"
+     * })
+     * @Form\Type("\Common\Form\Elements\InputFilters\ActionButton")
+     */
+    public $close = null;
+
+    /**
+     * @Form\Attributes({
+     *     "data-module": "govuk-button",
+     *     "type": "submit",
```

Comment:
2 buttons with type submit might be an accessibility issue

Reply:
All situations with a cancel/close and submit button I've found so far all use type submit with and id of cancel: there's a lot!

Reply:
Changed to cancel but the FE forces it to submit - one to work out why!